checks.go

```diff
@@ -12,35 +12,21 @@ import (
 	"reflect"
 	"regexp"
 	"strings"
+
+	"github.com/safinsingh/aeaconf2"
 )
 
-// check is the smallest unit that can show up on a scoring report. It holds all
-// the conditions for a check, and its message and points (autogenerated or
-// otherwise).
-type check struct {
-	Message string
-	Hint    string
-	Points  int
-
-	Fail         []cond
-	Pass         []cond
-	PassOverride []cond
-}
+var funcRegistry map[string]reflect.Type
```

Comment:
You can init this in var with type inference eg `var funcRegistry = make(map[string]reflect.Type)`. Having a map to a reflect type smells really bad though LOL I'll have to understand the code